src/providers/bitbucket.ts

```diff
@@ -0,0 +1,40 @@
+import { OAuthConfig, OAuthUserConfig } from "./oauth"
+
+interface BitbucketProfile {
+  username: string
+  account_id: string
+  display_name: string
+  links: {
+    avatar: {
+      href: string
+    }
+  }
+}
+
+export default function Bitbucket<P extends Record<string, any> = BitbucketProfile>(
+  options: OAuthUserConfig<P>
+): OAuthConfig<P> {
+  return {
+    id: "bitbucket",
+    name: "Bitbucket",
+    type: "oauth",
+    authorization: {
+      url: "https://bitbucket.org/site/oauth2/authorize",
+      params: {
+        scope: ""
+      }
+    },
+    token: "https://bitbucket.org/site/oauth2/access_token",
+    userinfo: "https://api.bitbucket.org/2.0/user",
+    profile(profile) {
+      return {
+        id: profile.account_id,
+        email: null,
+        image: profile.links.avatar.href,
+        name: profile.display_name
+      }
+    },
+    checks: ["none"],
```

Comment:
So `["state"]` does not work either?

Reply:
In my early tests, it doesn't, but it seems to be working now. I will remove this.